src/Stratis.Bitcoin.Features.RPC/RPCClient.cs

```diff
@@ -1139,16 +1139,18 @@ public Task<Transaction> DecodeRawTransactionAsync(byte[] raw)
         /// <summary>
         /// getrawtransaction only returns on txn which are not entirely spent unless you run bitcoinq with txindex=1.
         /// </summary>
-        /// <param name="txid"></param>
-        /// <returns></returns>
-        public Transaction GetRawTransaction(uint256 txid, bool throwIfNotFound = true)
+        /// <param name="txid">The transaction id to retrieve.</param>
+        /// <param name="blockHash">The optional block hash in which to look.</param>
+        /// <param name="throwIfNotFound">Whether to throw an exception if not found.</param>
+        /// <returns>The transaction </returns>
+        public Transaction GetRawTransaction(uint256 txid, uint256 blockHash = null, bool throwIfNotFound = true)
         {
-            return GetRawTransactionAsync(txid, throwIfNotFound).GetAwaiter().GetResult();
+            return GetRawTransactionAsync(txid, blockHash, throwIfNotFound).GetAwaiter().GetResult();
         }
 
-        public async Task<Transaction> GetRawTransactionAsync(uint256 txid, bool throwIfNotFound = true)
+        public async Task<Transaction> GetRawTransactionAsync(uint256 txid, uint256 blockHash = null, bool throwIfNotFound = true)
```

Comment:
See comment above, if you decide to change above - then change here makes sense too.